.github/workflows/ci.yaml

```diff
@@ -14,6 +14,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
       - name: Install Dependencies
-        run: npm ci --ignore-scripts
```

Comment:
nit: is there a reason why we're swapping to Yarn?

It may be nice to be consistent with either NPM or Yarn or etc. across LoopBack repos.

Removing `package-lock.json` will also break the website script that pulls this repo:

https://github.com/loopbackio/loopback.io/blob/edcd25d26fe07cf3b512a1b32cf9496a2ad956d6/update-blog.sh#L112-L122

Reply:
ya.. i think it's better to use npm for consistency and not breaking loopback.io once deploy. I totally forgot about it. Switching back. Thanks!